doc/10-icinga-template-library.md

```diff
@@ -5789,8 +5789,11 @@ ssl_cert_disable_ssl_versions | **Optional.** Disable specific SSL versions out
 ssl_cert_cipher               | **Optional.** Cipher selection: force {ecdsa,rsa} authentication.
 ssl_cert_ignore_expiration    | **Optional.** Ignore expiration date.
 ssl_cert_ignore_ocsp          | **Optional.** Do not check revocation with OCSP.
+ssl_cert_ignore_ocsp_errors   | **Optional.** Continue if the OCSP status cannot be checked
+ssl_cert_ignore_ocsp_timeout  | **Optional.** Ignore OCSP result when timeout occurs while checking
 ssl_cert_ignore_sct           | **Optional.** Do not check for signed certificate timestamps.
 ssl_cert_ignore_tls_renegotiation  | **Optional.** Do not check for renegotiation.
+ssl_cert_ignore_host_cn       | **Optional.** Do not complain if the CN does not match
```

Comment:
I wonder why the ordering differs between the changes in both files. So far, at least the ignore options seem to be ordered alphabetically, so I'd keep it this way. Also, most (unfortunately not all) descriptions end with a period, so I'd keep most of them consistent and add them here as well.

Reply:
In the web.conf it was a try to keep the order of the output of `check_ssl_cert  -h` version 2.40.0.
The descriptions are also taken from this help output and there is no period.

Reply:
> In the web.conf it was a try to keep the order of the output of `check_ssl_cert -h` version 2.40.0.

The order in `web.conf` is fine. I suggest to use the same in `10-icinga-template-library.md`.

> The descriptions are also taken from this help output and there is no period.

So is the description for `--ignore-ocsp` which is also consistent with the [majority of options in that file](https://github.com/Icinga/icinga2/blob/f7890247cc2d247cde503ec25389d77fe3052bf5/doc/10-icinga-template-library.md), so I opt for adding the period.

Reply:
like [0ae3625]?